src/main/java/bio/terra/common/ValidationUtils.java

```diff
@@ -6,8 +6,6 @@
 
 public final class ValidationUtils {
 
-    private static final String VALID_NAME_REGEX = "[a-zA-Z0-9_]{1,63}";
```

Comment:
This pattern didn't exactly match the one specified in the OpenAPI spec. Swagger now auto-generates server-side validation of any patterns specified in the spec, so there's no need for this any more.